account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -169,10 +169,16 @@ def create_company(self):
     def taxes_by_company(self, company_id, match_tax_ids):
```

Comment:
The TODO is because `match_tax_ids` is a list, which is a non hashable type. We have solved it in later versions passing the recordset, which is hashable (and there's no problem to have a recordset as argument. The problem is to return a recordset).

```suggestion
    def taxes_by_company(self, company_id, match_taxes):
```

Reply:
Done